### ISSUE TITLE
[BUG] Could not start application propcheck

### ISSUE THREAD

**Issue by squirmy** [ahamez/protox#201]
**Describe the bug**
Thanks for making a release :)

After updating to 1.7.6 I now get this error when running my app:

```bash
** (Mix) Could not start application propcheck: could not find application file: propcheck.app
````

Is there additional configuration I need to specify?

**To Reproduce**
Steps, code or protobuf messages to reproduce the behavior:

1. mix deps.update protox
2. mix phx.server

**Expected behavior**

Phoenix server starts.

**Environment (please complete the following information):**
Hex:    2.1.1
Elixir: 1.17.3
OTP:    25.3.2.13

**Comment by ahamez** [comment]
🤔 It seems I got things mixed up with the key `extra_applications`.

Can you try with `{:protox, github: "ahamez/protox", ref: "173ea1a"}` (you should erase the `deps` directory before)? 

On my side, with the following `mix.exs`, it works when running `mix phx.server`:
```
defmodule Foo.MixProject do
  use Mix.Project

  def project do
    [
      app: :foo,
      version: "0.1.0",
      elixir: "~> 1.17",
      start_permanent: Mix.env() == :prod,
      deps: deps()
    ]
  end

  # Run "mix help compile.app" to learn about applications.
  def application do
    [
      extra_applications: [:logger]
    ]
  end

  # Run "mix help deps" to learn about dependencies.
  defp deps do
    [
      {:protox, github: "ahamez/protox", ref: "173ea1a"},
      {:phoenix, "~> 1.7"}
    ]
  end
end
```

**Comment by squirmy** [comment]
Yep, that works perfectly.

Thanks!

**Comment by ahamez** [comment]
Great! It's now published in [1.7.7](https://hex.pm/packages/protox/1.7.7) :-)
